src/Components/Product.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+function Product({productLabel, productImage, imageTitle, productName, productPrice}) {
+
+
+
+    return(
+        <article>
+            <span>{productLabel}</span>
+            <img src={productImage} alt={imageTitle}/>
+            <p>{productName}</p>
+            <h4>{productPrice}</h4>
```

Comment:
Product price was wenselijker om alleen de cijfers in het variabel te zetten, de euroteken en de comma dus niet. Dan hoeven we alleen maar de cijfer mee te geven.